quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -173,6 +173,12 @@ struct Params {
     /// Determines whether app limited rounds with no bandwidth growth count
     /// towards the rounds threshold to exit startup.
     ignore_app_limited_for_no_bandwidth_growth: bool,
+
+    /// Initial pacing rate for a new connection before an RTT
+    /// estimate is available.  This rate serves as an upper bound on
+    /// pacing rate calculated by dividing the initial cwnd by the
+    /// first RTT estimate.
+    initial_pacing_rate_bytes_per_second: Option<u64>,
```

Comment:
> This rate serves as an upper bound on
> pacing rate calculated by dividing the initial cwnd by the
> first RTT estimate.

Might be missing something but how is this enforcing an upper bound? It seems to override the value based on my read of the code below.

```
    if let Some(pacing_rate) = params.initial_pacing_rate_bytes_per_second {
        return Bandwidth::from_bytes_per_second(pacing_rate);
    }
```

Reply:
See https://github.com/cloudflare/quiche/pull/2115#discussion_r2237998067